model/userdata.go

```diff
@@ -147,3 +148,105 @@ func validateCoreosCloudInit(content []byte) error {
 func validateNone([]byte) error {
 	return nil
 }
+
+func (self UserDataPart) Base64UserDataRef() (string, error) {
+	// Change this to Base64IgnitionConfigRef to test with ignition
+	return self.Base64BashScriptRef()
```

Comment:
Ignition is not used by default for now. However, I like to leave the ignition support able to be enabled in code so that we can start testing with it toward #728 🤓